src/components/document/document-workspace.tsx

```diff
@@ -44,15 +44,18 @@ export class DocumentWorkspaceComponent extends BaseComponent<IProps, {}> {
     );
   }
 
-  private async guaranteePrimaryDocument() {
-    const { appConfig: { defaultDocumentType, defaultDocumentContent },
+  private async guaranteeInitialDocuments() {
+    const { appConfig: { defaultDocumentType, defaultDocumentContent,
+                         defaultLearningLogDocument, defaultInitialLearningLogTitle },
             db, ui: { problemWorkspace } } = this.stores;
     if (!problemWorkspace.primaryDocumentKey) {
       const defaultDocument = await db.guaranteeOpenDefaultDocument(defaultDocumentType, defaultDocumentContent);
       if (defaultDocument) {
         problemWorkspace.setPrimaryDocument(defaultDocument);
       }
     }
+    // Guarantee the user starts with one learning log
+    defaultLearningLogDocument && await db.guaranteeLearningLog(defaultInitialLearningLogTitle);
```

Comment:
```suggestion
    defaultLearningLogDocument &&
      await db.guaranteeLearningLog(initialLearningLogTitle || defaultLearningLogTitle);
```
Without this, if you specify `defaultLearningLogDocument: true` but don't specify an `initialLearningLogTitle` (for instance if you don't care to have the initial title be different than the default title) you get an empty string for the title.

Reply:
Oh! Thank you for catching that, fixed!